### PR TITLE
Enhancement/1/mn

### PR DESCRIPTION
**Pull request by maxnixon**

### CODE REVIEW

File: landingPage.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title></title>
```
Comment (robcapell) [inline landingPage.html]: Use the title tag, to appropriately name this page.

File: productDetailPage.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title></title>
```
Comment (robcapell) [inline productDetailPage.html]: Use the title tag, to appropriately name this page.

File: home.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (robcapell) [inline home.html]: What is this page to be used for? Since we already have a landingPage.html and productDetailPage.html?

Comment (maxnixon) [inline home.html]: that was the original name I gave the file.. the system probably got confused after I renamed it.

File: productDetailPage.html (hunk 2 above)
Comment (maxnixon) [inline productDetailPage.html]: okay

File: landingPage.html (hunk 1 above)
Comment (maxnixon) [inline landingPage.html]: okay

**Pull request merged**